scripts/deploy.ts

```diff
@@ -0,0 +1,16 @@
+import { ethers } from "hardhat";
+import { TIMELOCK_DURATION } from "../test/constants";
+
+async function main() {
```

Comment:
This wasn't explicitly stated in the AC - so wouldn't worry about it and since this contract suite is going to relatively simple it probably doesn't matter

but, for this I would recommend we use a HH deploy script (example here - https://github.com/shapeshift/fox-staking-3/blob/develop/deploy/core/stakingRewardsFactory.ts)

It makes testing easier and requires less boilerplate.  More of an FYI than anything else.